common/util.go

```diff
@@ -120,6 +121,26 @@ func GetCurrentUser() (uint32, uint32, error) {
 		return 0, 0, err
 	}
 
+	if runtime.GOOS == "windows" {
```

Comment:
I am not sure that this is needed anymore with my latest pull request. Since we don't have the Windows service launching the programs, then WinFSP should always know who the current user.

Reply:
From our discussions, it sounds like this is the easiest path forward.
I'll leave this be and resolve this.